### PR TITLE
global hub troubleshooting consistent with rhacm troubleshooting

### PR DESCRIPTION
**Pull request by jc-berger**
Jira: https://issues.redhat.com/browse/ACM-12306

### CODE REVIEW

File: troubleshooting/trouble_global_hub_database_dump_restore.adoc [hunk 1]
```diff
@@ -1,10 +1,15 @@
-[#gh-database-and-dump-restore]
-= Using the database dump and restore for troubleshooting
+[#troubleshooting-database-dump-restore]
```
Comment (swopebe) [inline troubleshooting/trouble_global_hub_database_dump_restore.adoc:1]: I don't know why we went down the road of putting the whole t-shooting word in these identifiers, but i see that many of them have them. It's not at all necessary.

File: troubleshooting/trouble_global_hub_compliance_data.adoc [hunk 2]
```diff
@@ -1,7 +1,17 @@
...
+= Troubleshooting compliance data 
 
-The Grafana Datasource is mainly from a table named `history.local_compliance`. Its records are generated by a summarization routine that starts nightly at 00:00:00. You do not typically need to run the summary process manually. In some cases, unexpected errors might occur when running the compliance job, so it is necessary to manually log in to the database to run the entire summary process to recover the data that is not generated. You can follow the procedure in link:../global_hub/global_hub_compliance.adoc#global-hub-compliance-manual[Running the summarization process manually] to recover the data.
+The Grafana Datasource is mainly from a table named `history.local_compliance`. Its records are generated by a summarization routine that starts nightly at 00:00:00. You do not typically need to run the summary process manually. 
+
+[#symptom-errors-compliance-job]
+== Symptom: Errors when running the compliance job
+
+In some cases, unexpected errors might occur when running the compliance job, so it is necessary to manually log in to the database to run the entire summary process to recover the data that is not generated.
```
Comment (swopebe) [inline troubleshooting/trouble_global_hub_compliance_data.adoc]: `so it is necessary to manually log...` should just be: you need to manually log.

"It is necessary" is not concise, extra words that are not needed. I realize you didn't add this, just trying to get us more concise across the board.

**Review by swopebe — COMMENTED** [review]
See one comment @jc-berger before merging but otherwise looks very good. TY so much for making things consistent.

**Review by swopebe — APPROVED** [review]

**Comment by ldpliu** [comment]
@yanmxa Could you help to review all the globalhub troubleshooting doc
1. something looks too old.
For example: https://github.com/stolostron/rhacm-docs/blob/2.11_stage/troubleshooting/trouble_access_postgres.adoc
2. I am nost sure if this file still needed, https://github.com/stolostron/rhacm-docs/blob/6b9492ce67bcfc0b459564a63c1f7aa6bfc84a1c/troubleshooting/trouble_global_hub_compliance_data.adoc

**Review by ldpliu — APPROVED** [review]

**Comment by yanmxa** [comment]
Hey @jc-berger ! I draf another pr to modify the database access based on the current change: https://github.com/stolostron/rhacm-docs/pull/6594. Please feel free to contact me if you have any confusion about it.

**Pull request merged**